### PR TITLE
Add beehive inspector

### PR DESCRIPTION
**Pull request by misode**
Adds two lines of lore to beehives describing the contents of it.
image image

### CODE REVIEW

File: gm4_beehive_inspector/data/gm4_beehive_inspector/loot_tables/lore.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:stick",
+          "functions": [
+            {
+              "function": "minecraft:set_lore",
+              "lore": [
+                [{"text":"Bees: ","color":"gold","italic":false},{"score":{"objective":"gm4_beehive_insp","name":"#bees"},"color":"yellow"}],
+                [{"text":"Honey: ","color":"gold","italic":false},{"nbt":"Item.tag.BlockStateTag.honey_level","entity":"@s","color":"yellow"}]
```
Comment (AndanteDevs) [inline gm4_beehive_inspector/data/gm4_beehive_inspector/loot_tables/lore.json]: Just lang :)

```suggestion
                [{"translate":"%1$s%3427655$s","with":["Bees",{"translate":"text.gm4.beehive_inspector.bees"}],"color":"gold","italic":false},": ",{"score":{"objective":"gm4_beehive_insp","name":"#bees"},"color":"yellow"}],
                [{"translate":"%1$s%3427655$s","with":["Honey",{"translate":"text.gm4.beehive_inspector.honey"}],"color":"gold","italic":false},": ",{"nbt":"Item.tag.BlockStateTag.honey_level","entity":"@s","color":"yellow"}]
```

Comment (misode) [reply gm4_beehive_inspector/data/gm4_beehive_inspector/loot_tables/lore.json]: Oh right, I forgot about that

**Comment by ToffeeMax** [comment]
@SpecialBuilder32  Would you feel better if it was made into like a "tool tip" module for things that don't usually have a tool tip, but would be useful?

**Comment by SandSander** [comment]
**This module is quite handy and small!**
This module is already on the public server, 
Tested and works fine! :)

**Comment by SandSander** [comment]
This module seems to work? Don't know what the plans are for this module?
Maybe @ToffeeMax his idea seemed like a good idea?

**Comment by Bloo-dev** [comment]
I don't mind this being simple as it is really helpful.

We could call it Ambrosia and disable natural regeneration.

**Comment by SandSander** [comment]
**That's a great idea!**

**Comment by misode** [comment]
Should this also work for bee nests?

**Comment by SandSander** [comment]
Maybe yes?

**Comment by jpeterik12** [comment]
I thought it was chosen only to work on hives to give hives a purpose other than just asthetics

**Pull request merged**